modules/api.js

```diff
@@ -0,0 +1,91 @@
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
+
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+const baseEndPoint = 'games/';
+const scoresEndPoint = '/scores/';
+const id = 'AZwV69nnEThqVXPyn';
+
+const player = {
+  user: String,
+  score: Number,
+};
+
+const title = {
+  name: 'valleyball',
+};
+
+export default class Game {
+    static createNewGame = async () => {
+      try {
+        const response = await fetch(baseUrl + baseEndPoint, {
+          method: 'POST',
+          headers: {
+            'content-type': 'application/json',
+          },
+          body: JSON.stringify(title),
+        });
+        const data = await response.json();
+        return data;
+      } catch (error) {
+        return error;
+      }
+    }
+
+    static createNewScore = async () => {
+      if (scoreInput.value.length !== 0 && nameInput.value.length !== 0) {
+        player.user = nameInput.value;
+        player.score = scoreInput.value;
+      }
+      try {
+        const response = await fetch(baseUrl + baseEndPoint + id + scoresEndPoint, {
```

Comment:
[OPTIONAL]
- [ ] Since you do not have a lot of places where the API endpoints for score is being used, it would be more great for clarity and simplicity to not store it in some sort of variable after that you have some useless concatenation.
It would look simple to use the API endpoint this way.
`'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/AZwV69nnEThqVXPyn/scores/'`